examples/bri-3/src/bri/transactions/agents/transactionStorage.agent.ts

```diff
@@ -25,6 +48,8 @@ export class TransactionStorageAgent extends PrismaService {
       transactionModel.signature,
       transactionModel.status,
     );
+
+    //return this.autoMapper.map(transactionModel, Transaction, Transaction);
```

Comment:
use this instead of the above. TODO fields can be ignore in the profile or by avoiding @automap